.github/workflows/builds-and-tests.yml

```diff
@@ -205,6 +205,10 @@ jobs:
       env:
         VCPKG_DEFAULT_BINARY_CACHE: ${{ github.workspace }}/vcpkg_cache
       run: |
+        $PortFile = 'C:\vcpkg\ports\metis\portfile.cmake'
+        $OriginalURL = 'http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/metis-${METIS_VERSION}.tar.gz'
+        $NewURL = 'https://github.com/mfem/tpls/raw/gh-pages/metis-5.1.0.tar.gz'
```

Comment:
Do we want to use `${METIS_VERSION}` here instead of hard-coding version 5.1.0?

Reply:
We can keep it simple and keep the hardcoded version, as it's not easy to find where this `${METIS_VERSION}` comes from: the day it'll change, we'll have to add it in `tpls` and the jobs will fail.

Reply:
Okay, that actually sounds better, scratch my previous response.